tests/__init__.py

```diff
@@ -19,7 +19,4 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-"""Pytest fails without this file in some cases.
-
-Fixes https://gitlab.com/nekokatt/hikari/-/issues/408.
```

Comment:
We should probably keep the issue fixed so people actually know what is being fixed - moving to a different repo hosting thing doesn't mean we should necessarily throw out old (but maybe relevant) info.